arduino/smartcar/vehicle_config.json

```diff
@@ -1,4 +1,86 @@
 {
```

Comment:
I think we removed this file in a previous PR, was adding it a mistake or is there a reason that it's back?

Reply:
Oh okay I see, my bad.